codecov_cli/services/upload/upload_sender.py

```diff
@@ -53,19 +52,7 @@ def send_upload_data(
             "version": codecov_cli_version,
             "ci_service": ci_service,
         }
-
-        # Data to upload to Codecov
-        pull_dict = (
-            get_pull(git_service, slug, pull_request_number) if not token else None
-        )
-
-        if is_fork_pr(pull_dict):
-            headers = {
-                "X-Tokenless": pull_dict["head"]["slug"],
-                "X-Tokenless-PR": pull_request_number,
-            }
-        else:
-            headers = get_token_header_or_fail(token)
+        headers = get_token_header(token)
```

Comment:
Why are some commands using `get_token_header` vs `get_token_header_or_fail`? How do we know when to use which?

Reply:
`get_token_header` is for supporting `tokenless`, if someone tries to upload using tokenless `get_token_header_or_fail` just fails and prevents them from doing anything

Reply:
Ahh that makes sense, worth leaving another comment here? 😅